### ISSUE TITLE
Extended Support Release (ESR) docs

### ISSUE THREAD

**Issue by torgeirl** [mattermost/docs#2665]
On April 14, 2019 I received the following email from Mattermost Support:
> Extended Support Releases or ESRs are intended for organizations who value stability over having the newest features and improvements, or for organizations who have a long internal testing and certification process to undergo when upgrading.
> 
> We have been maintaining 4.10 as our current ESR.  This version is coming to the end of its lifecycle and we will be implementing version 5.9 as a new ESR. Starting 7/15/2019 we will no longer be backporting fixes to 4.10.  Please upgrade your systems to 5.9 l as soon as possible to continue to receive important bug and security updates.
> 
> To receive future updates about Extended Support Releases, sign up for our mailing list here.

**tl;dr:** 5.9 is the new ESR, support for 4.10 ends on July 15.

These updates are currently not included in the documentation, and I haven't seen this announced anywhere else either (official Twitter, official blog). At an absolute minimum the docs should be updated to prevent someone from setting up a server now using 4.10.

More generally, I would encourage the team to make an ESR roadmap that details the planned ESR release cycle going forward. A good example can be found at the [Django Project version page](https://www.djangoproject.com/download/).

**Comment by amyblais** [comment]
Twitter and blog announces sound good too, @wiersgallak is that something you can help coordinate with marketing or let me know if you need any help with that?

**Comment by torgeirl** [comment]
My suggested changes got approved and have been merged to `master` so the immediate nature of this issue has been resolved. :+1: 

> Twitter and blog announces sound good too

@amyblais: great! Should we keep this issue open while that's been prepared or for any broader   discussion a roadmap/improved ESR comms in general, or would you like to keep that internally?

**Comment by justinegeffen** [comment]
@amyblais, imo this has been addressed and resolved. Wanted to confirm with you before I close the issue though.

**Comment by torgeirl** [comment]
@justinegeffen: the roadmap etc found in [current ESR docs](https://docs.mattermost.com/administration/extended-support-release.html) is exactly what I had in mind when I opened this issue. :heart_eyes:

**Comment by justinegeffen** [comment]
Thanks, @torgeirl! May I close this as resolved?

**Comment by torgeirl** [comment]
@justinegeffen: I assume @amyblais has any future work on this tracked elsewhere, and that this now can be closed. :+1:
